pkg/build/nodeimage/internal/kube/builder_docker.go

```diff
@@ -98,6 +90,15 @@ func (b *dockerBuilder) Build() (Bits, error) {
 		"cmd/kubelet",
 	}
 
+	// make sure we cd back when done
```

Comment:
maybe we should do this right after source version, just in case the rest gets refactored?

Reply:
You're call, but I recommended moving it here since this is in close proximity to where it is actually needed. So I think that might help reduce the risk of a refactor causing issues since it's more "visible" if someone is touching the make call.

`-C` seems reasonable too.

Reply:
> You're call, but I recommended moving it here since this is in close proximity to where it is actually needed. 

That makes sense, IMHO it's "needed" by the rest of the code, but really it's not "needed",  I can't recall why we didn't just use `-C` ... but this code is an old amalgam at this point, and I don't see any reason we couldn't do that now and drop the chdir altogether.

Reply:
@oduludo I'd be open merging as-is, or if you want to take a stab at moving to `-C` we'll try that :-)
/hold
Will check back tomorrow